src/plugins/intel_cpu/src/emitters/utils.cpp

```diff
@@ -131,16 +131,19 @@ const char * RegPrinter::get_name(REG_T reg, const char *name) {
     if (name == nullptr) {
         return reg_name;
     } else {
+        constexpr size_t cnt = 1024;
```

Comment:
It seems like 1024 is an experimental or guess number to avoid flush. I am wondering if it is possible to use an shared mutex across all print,  lock inside print function to serialize each print pipeline. Until previous print finish, we start new print.

Reply:
Thanks Chenhu for the commet! I've tried to use mutex mechanism inside print function (also tried that in the scope of h.call and get_name functions), it doesn't fix the issue. It seems not an issue of synchronism. Further discussion is always welcomed.